### DISCUSSION TITLE
Secret key appears to be invalid

### DISCUSSION THREAD

**Discussion by TamirErez** [Erudika/scoold#308] [Q&A]
Hi,
I have started a para server locally and created an app called "scoold", and copied the keys to the scoold conf file.
When I try to run the scoold server I get the following error;
![image](https://user-images.githubusercontent.com/15075254/170991988-74a988f7-339c-43da-b272-c70da0206fdc.png)
As well as on the para server itself:
![image](https://user-images.githubusercontent.com/15075254/170992070-7f282dc2-436b-4190-b7df-b8802b0c24d6.png)
This is my application.conf file:
![image](https://user-images.githubusercontent.com/15075254/170992128-444a6cf3-6b63-4e64-9567-b7763aa08906.png)
Any help would be appreciated

**Comment by albogdano** [comment]
Hi, your configuration file syntax is invalid and it's being rejected. Please remove all curly braces `{}` and try again. The file format is HOCON.

**Comment by TamirErez** [comment]
Thank you for the reply!
Unfortunately it didn't help :(
With this config file:
![image](https://user-images.githubusercontent.com/15075254/171091925-1056db56-08ac-46a3-b2eb-a2e9eff7bf1a.png)
I got the same error:
![image](https://user-images.githubusercontent.com/15075254/171091953-a3bf9c59-4b1b-4336-8404-09d3b1603d6c.png)

Also, I debugged the config file to try and understand and I noticed System.getProperty in the Config.java file always returns null:

![image](https://user-images.githubusercontent.com/15075254/171095647-64ac7476-a790-42c3-a55a-4323c4ceaef0.png)

Btw I run my ScooldServer with the setting -Dconfig.file=application.conf 

I am able to send valid requests via postman to para

**Comment by TamirErez** [comment]
I finally managed to get it working by manually pasting the app secret key into the paraSecretKey method as the default value.
Which leaves the question why is the configuration not loaded from the file?
Thanks
